gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -275,12 +275,26 @@ private WriteObjectResponse doResumableUpload() throws IOException {
 
       boolean objectFinalized = false;
       while (!objectFinalized) {
+        if (responseObserver.isComplete()) {
```

Comment:
Why are we combining writechannel write fix with this?

Reply:
Rebased wrong branch. Will remove.